packages/components/src/InputSelect/Component.purs

```diff
@@ -0,0 +1,99 @@
+module KSF.InputSelect.Component where
+
+import Prelude
+
+import Data.Array (snoc)
+import Data.Foldable (foldMap)
+import Data.Generic.Rep (class Generic)
+import Data.Generic.Rep.Show (genericShow)
+import Data.Maybe (Maybe(..), fromMaybe, isJust, isNothing)
+import Data.String (toLower)
+import Effect (Effect)
+import React.Basic (JSX)
+import React.Basic as React
+import React.Basic.DOM as DOM
+import React.Basic.DOM.Events (targetChecked)
+import React.Basic.Events (handler)
+
+type Self = React.Self Props State
+
+type Props =
+  { type_           :: InputType
+  , name            :: String
+  , value           :: Maybe String
+  , checked         :: Maybe Boolean
+  , onChange        :: Maybe Boolean -> Effect Unit
+  , label           :: Maybe String
+  , required        :: Maybe Boolean
+  }
+
+type State =
+  { checked :: Boolean }
+
+data InputType = Radio | Checkbox
```

Comment:
I guess the idea is to use this component also for the `Radio` inputs we now use via `InputField` component, right? Why did you decide to separate this (and `Radio`) from the `InputField` component?

Reply:
If I change the InputField component to support checkboxes that requires adding a prop for checked that is totally useless but still required to be defined for all other input types. As radio buttons and checkboxes have similar uses It makes sense to separate them from the rest of the input types. The Radio part of this component is not implemented fully yet so I might just remove it from here and it could be added if we start using radio buttons more in the future.